### ISSUE TITLE
Erro Theme

### ISSUE THREAD

**Issue by fuedgabriel** [januwA/flutter_video_box#15]
Performing hot reload...
Syncing files to device Custom Phone 1...

Compiler message:
/C:/SRC/flutter/.pub-cache/hosted/pub.dartlang.org/video_box-0.10.0/lib/widgets/buffer_slider.dart:101:27: Error: The getter 'bodyText1' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('/C:/SRC/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'bodyText1'.
          theme.textTheme.bodyText1.copyWith(
                          ^^^^^^^^^

**Comment by januwA** [comment]
The `api` has changed on the` master` branch of `flutter`

**Comment by fuedgabriel** [comment]
Launching lib\main.dart on C14CR21 in debug mode...
Running Gradle task 'assembleDebug'...

Compiler message:
/C:/SRC/flutter/.pub-cache/hosted/pub.dartlang.org/video_box-0.10.0/lib/widgets/buffer_slider.dart:101:27: Error: The getter 'bodyText1' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('/C:/SRC/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'bodyText1'.
          theme.textTheme.bodyText1.copyWith(
                          ^^^^^^^^^
Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
build failed.

FAILURE: Build failed with an exception.

* Where:
Script 'C:\SRC\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 780

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\SRC\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 23s
Finished with error: Gradle task assembleDebug failed with exit code 1





-----------------------------------------------------------------------------------------
The error I was giving was the one above but I managed to resolve it by editing the source code.
I added in the file 'package: flutter / src / material / text_theme.dart'
Within the 'TextTheme' class the value this.bodyText1.
Then I added 'final TextStyle bodyText1;'
